nuxt.config.js

```diff
@@ -60,8 +60,12 @@ export default defineNuxtConfig({
     ],
     runtimeConfig: {
         public: {
+            // siteUrl: process.env.NUXT_PUBLIC_SITE_URL,
+            siteUrl: process.env.NUXT_PUBLIC_SITE_URL,
```

Comment:
This could be :
```suggestion
            siteUrl: localeDomains.getDomainByLocale(process.env.DEFAULT_LOCALE),
```

---

```javascript
// locale-domains.js

module.exports = [
  { code: 'en-gb', name: 'UK', domain: process.env.DOMAIN_UK },
  { code: 'en-sg', name: 'SG', domain: process.env.DOMAIN_SG },
  { code: 'en-ie', name: 'IE', domain: process.env.DOMAIN_IE },
  { code: 'en-au', name: 'AU', domain: process.env.DOMAIN_AU },
  { code: 'en-us', name: 'US', domain: process.env.DOMAIN_US },
  { code: 'pl', name: 'PL', domain: process.env.DOMAIN_PL }
];

function getDomainByLocale(locale) {
  const localeObj = module.exports.find(obj => obj.code === locale);
  return localeObj ? localeObj.domain : null;
}

module.exports.getDomainByLocale = getDomainByLocale;

```
After which, we can just get rid of `NUXT_PUBLIC_SITE_URL` right?